backend/open_webui/routers/openai.py

```
@@ -600,6 +627,10 @@ async def generate_chat_completion(
     user=Depends(get_verified_user),
     bypass_filter: Optional[bool] = False,
 ):
+    import time
```

Ruff (0.12.2)

Redefinition of unused `time` from line 5

Remove definition: `time`

(F811)